src/core/triggers.systemd.in

```diff
@@ -69,3 +70,90 @@ if posix.access("%{_localstatedir}/lib/rpm-state/systemd/needs-reload") then
         posix.wait(pid)
     end
 end
+
+%transfiletriggerin -P 100700 -p <lua> -- @sysusersdir@
+-- This script will process files installed in sysusers.d to
+-- create specified users automatically. The priority is set
+-- such that it will run before the tmpfiles file trigger.
+if posix.access("/run/systemd/system") then
+    pid = posix.fork()
+    if pid == 0 then
+        assert(posix.exec("%{_bindir}/systemd-sysusers"))
+    elseif pid > 0 then
+        posix.wait(pid)
+    end
+end
+
+%transfiletriggerin -P 100500 --  @tmpfilesdir@
+-- This script will process files installed in tmpfiles.d to
+-- create tmpfiles automatically. The priority is set
+-- such that it will run after the sysusers file trigger,
+-- but before any other triggers.
+if posix.access("/run/systemd/system") then
+    pid = posix.fork()
+    if pid == 0 then
+        assert(posix.exec("%{_bindir}/systemd-tmpfiles", "--create"))
+    elseif pid > 0 then
+        posix.wait(pid)
+    end
+end
+
+%transfiletriggerin --  @udevhwdbdir@
+-- This script will automatically invoke hwdb update if files
+-- have been installed or updated in hwdb.d.
+if posix.access("/run/systemd/system") then
+    pid = posix.fork()
+    if pid == 0 then
+        assert(posix.exec("%{_bindir}/systemd-hwdb", "update"))
+    elseif pid > 0 then
+        posix.wait(pid)
+    end
+end
+
+%transfiletriggerin --  @catalogdir@
+-- This script will automatically invoke journal catalog update if files
+-- have been installed or updated in the catalog.
```

Comment:
Here too, I think this should be explicit that only `@catalogdir@` is monitored.